xcdat/bounds.py

```diff
@@ -133,24 +141,30 @@ def add_missing_bounds(self, width: float = 0.5) -> xr.Dataset:
         axes = CF_NAME_MAP.keys()
 
         for axis in axes:
-            coord_var = None
+            # Check if the axis coordinates can be mapped to.
+            try:
+                get_axis_coord(self._dataset, axis)
+            except KeyError:
+                continue
 
+            # Determine if the axis is also a dimension by determining if there
+            # is overlap between the CF axis names and the dimension names. If
+            # not, skip over axis for validation.
+            if len(set(CF_NAME_MAP[axis]) & set(self._dataset.dims.keys())) == 0:
+                continue
+
+            # Check if bounds also exist using the "bounds" attribute.
+            # Otherwise, try to add bounds if it meets the function's criteria.
             try:
-                coord_var = get_axis_coord(self._dataset, axis)
+                self.get_bounds(axis)
```

Comment:
```suggestion
                self.get_bounds(axis)
                continue
```

I forgot to add `continue` if bounds are found.